site/src/hooks/useClipboard.test.tsx

```diff
@@ -1,3 +1,22 @@
+import { act, renderHook } from "@testing-library/react";
+import { GlobalSnackbar } from "components/GlobalSnackbar/GlobalSnackbar";
+import { ThemeProvider } from "contexts/ThemeProvider";
+import {
+  type UseClipboardInput,
+  type UseClipboardResult,
+  useClipboard,
+} from "./useClipboard";
+
+describe(useClipboard.name, () => {
+  describe("HTTP (non-secure) connections", () => {
+    scheduleClipboardTests({ isHttps: false });
+  });
+
+  describe("HTTPS (secure/default) connections", () => {
+    scheduleClipboardTests({ isHttps: true });
+  });
+});
```

Comment:
I'm a fan of consolidating the files as long as nothing breaks, but one of the issues I had was that if I had all the test cases in the same file, any call to `userEvent.setup` basically "polluted" the global scope with a mock clipboard that would always succeed no matter what

As in, there would be cases where code would succeed when it shouldn't. (The big one that comes to mind is that the clipboard would be available on the HTTP-only tests)

Just making sure: is that still a concern for this test file, or does the updated `jest.setup` file take care of that now?

Reply:
Yeap, since we now are using `restoreAllMocks` and doing things inside of the `before/after` each scope, it is working fine 👍

Reply:
You're a hero, Bruno!